### PR TITLE
[Paperwork] adds Internal affairs agents from paradise.

### PR DESCRIPTION
**Pull request by QuacksQ**
## About The Pull Request
Adds Internal Affairs agents as a new command job with 2 slots to the game.
Internal affairs agents are tasked with making sure SOP (Standard Operating Procedure) is followed on station.

## How Does This Help ***Gameplay***?
New job for people to play around with and this is also the perfect job for paperwork mains.

## How Does This Help ***Roleplay***?
IAA is tasked with making sure people follow SOP, failure to do so will result in a report or demotion etc etc, they generate roleplay basically and are a form of enforcing IC rules.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary> 

</details>

## Changelog

:cl:
add: Adds Internal affairs agents to the game.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -194,6 +194,10 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	name = "Cyborg"
 	icon_state = "Cyborg"
 
+/obj/effect/landmark/start/internal_affairs_agent
```
Comment (nevimer) [inline code/game/objects/effects/landmarks.dm:197]: move this to a better spot, not between ai and cyborg

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -190,6 +190,13 @@
 	greyscale_config = /datum/greyscale_config/encryptionkey_centcom
 	greyscale_colors = "#24a157#dca01b"
 
+/obj/item/encryptionkey/headset_internalaffairs
```
Comment (nevimer) [inline code/game/objects/items/devices/radio/encryptionkey.dm:193]: ditto

File: code/game/objects/items/devices/radio/headset.dm [hunk 3]
```diff
@@ -182,6 +182,11 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "srvsec_headset"
 	keyslot = new /obj/item/encryptionkey/headset_srvsec
 
+/obj/item/radio/headset/headset_internalaffairs
```
Comment (nevimer) [inline code/game/objects/items/devices/radio/headset.dm:185]: newline at the end for organizing better please, spacing entries

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 4]
```diff
@@ -384,3 +384,15 @@
 	icon_state = "pda-clear"
 	greyscale_config = null
 	greyscale_colors = null
+
+/**
+ * Internal Affairs Agent
```
Comment (nevimer) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:389]: move this after the warden and before engineering

**Review by nevimer — CHANGES_REQUESTED** [review]
just some better placement

**Review by nevimer — APPROVED** [review]
looks alright! There might be missed bits that sneak in, but we can sort those out when noticed.

**Review by nevimer — CHANGES_REQUESTED** [review]
Silver ID for them, as you are making them a part of command :P

**Comment by QuacksQ** [comment]
> Silver ID for them, as you are making them a part of command :P

image
Silver ID on there now.

**Review by nevimer — APPROVED** [review]
merging in an hour.

**Pull request merged**